### DISCUSSION TITLE
KowalewskiPawel

### DISCUSSION THREAD

**Discussion by LachPawel** [Git-Prove/GitProve-Projects#2] [Introductions]
Hey everyone,

I'm Pawel, and I'm thrilled to be a founder of the GitProve community! A little about me:

📍 Location: Warsaw/Poland

👨‍💻 Profession: Software Engineer

🛠️ Tech Stack: JavaScript, TypeScript, React, Node.js

🌱 Currently Learning: Expanding my general Backend and Dev-Ops knowledge

📚 Favourite Book: YDJS by Kyle Simpson

🎮 Hobbies: Motorsports, SimRally, Playing the Guitar and other instruments, Learning Foreign Languages, Gym and Bouldering

I joined GitProve to help in building the community, collaborate on exciting projects, and share my knowledge. I'm particularly interested in Web Development.

If anyone wants to collaborate, discuss tech, or just chat, feel free to reach out! I'm always up for a good conversation.

Looking forward to building, learning, and sharing with all of you!

Cheers,
Pawel

**Comment by pwierzbieniec** [comment]
Hey everyone,

I'm Patryk, and I'm thrilled to be a member of the GitProve community! A little about me:

📍 Location: Gdynia/Poland

👨‍💻 Profession: Software Engineer

🛠️ Tech Stack: JavaScript, React

🌱 Currently Learning: Expanding Typescript knowledge and GraphQL

📚 Favourite Book: Can't Hurt Me - David Goggins

🎮 Hobbies: RPG - admin of server Neverwinter Nights Enchanced Edition, culture of drinking tea, Gym and Traveling

I joined GitProve to collaborate on exciting projects, and share my knowledge. I'm particularly interested in Web Development.

If anyone wants to collaborate, discuss tech, or just chat, feel free to reach out! I'm always up for a good conversation.

Looking forward to building, learning, and sharing with all of you!

Cheers,
Patryk
